cargo-miri/bin.rs

```diff
@@ -694,6 +689,18 @@ fn phase_cargo_rustc(args: env::Args) {
     }
 }
 
+fn forward_patched_extern_arg(args: &mut impl Iterator<Item = String>, cmd: &mut Command) {
```

Comment:
Can you move this function up, to the other helper functions?

Reply:
(I copied this, including its location, and the bellow `extern_flag` change, from #1671 directly without modification, but sure I can change them.)